small fixes in input for calculate_indices() rgb()

GL-S opened the pull request:
### Proposed changes
added line to prevent warnings from dask.
Changed collection name in calculate_indices(), and reset band names in rgb() to default to avoid errors

### Closes issues (optional)
- Closes Issue #1271

### Checklist 
(Replace `[ ]` with `[x]` to check off)

- [X] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [X] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [ ] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)

GL-S committed:
small fixes in input for calculate_indices() rgb()

added line to prevent warnings from dask.
Changed collection name in calculate_indices(), and reset band names in rgb() to default to avoid errors

GL-S committed:
test

JM-GA commented on Beginners_guide/01_Jupyter_notebooks.ipynb:
```diff
@@ -60,7 +60,7 @@
     "\n",
     "To automatically run all cells in a notebook, navigate to the \"Run\" tab of the menu bar at the top of JupyterLab and select \"Run All Cells\" (or the option that best suits your needs).\n",
-    "When a cell is run, the cell's content is executed.\n",
+    "When a cell is run,the cell's content is executed.\n",
```
@GL-S - might need to revert this change - guessing it was the one testing git access

GL-S replied:
Yes you are correct, I will

GL-S committed:
revert last test commit

GL-S committed:
Revert "revert last test commit"
revert last test commit
This reverts commit 7d8d503ab704f94a1cb529085b032b896490822e.

GL-S committed:
revert test commit

This reverts commit adddf580c86fd96cdb274c7730ba9c7b7bcc75f8.

robbibt commented:
This looks great, thankls @GL-S! I left a few minor comments on the notebook... for the Dask thing, my main priority is just making sure the new line doesn't accidently cause any other downstream issues. If we're sure it doesn't, then we could consider moving it from this notebook to the `dea_tools.dask` module instead to hide it from external users?

GL-S commented:
Thank you for the comments @robbibt . 
The warnings appeared when running the model for classifying the satellite images, there is a lot of printing which doesn't look great.
 I added the line about the dask's warnings because it was a solution suggested by the warning itself (screenshot added below). I am actually not sure what it does in details, it should have an influence on how the processes run and exit. It's probably safer not to add it to `dea_tools.dask` yet. 
 ...maybe a `filterwarnings` in  `dea_tools.dask` is a better solution? It doesn't work when added to the 4_Classify_satellite_data.ipynb

![warnings dask](https://github.com/user-attachments/assets/4fdbf07c-4c19-461c-9ef9-32d458211cc3)

Regarding the comments about the band names, I get your point. I will remove those commented lines

robbibt commented:
Thanks @GL-S! Yeah, those warnings look terrible, it would definitely be nice to hide them away.

@emmaai Do you know what these Dask warnings in the screenshot above mean, and whether adding `dask.config.set({'distributed.worker.daemon': False})` to our `dea_tools.dask` script would be a safe move?

emmaai commented:
The warning seems to suggest there was a conflict between config worker as daemon and number of jobs = 1. Intuitively I don’t know why it can’t be both. The suggested solution is setting worker process as non-daemon. I don’t think it’s a good idea in general. Also as it runs with scikit-learn, I suspected the warnings actually suggested something different and more serious. Scikit-learn is multi-threading, it’s tricky to run with dask. I didn’t read the notebook. If it couldn’t be understood and resolved with reasonable thought, I’d suggest to keep it until further investigation.

GL-S committed:
product and collection names fixed

GL-S commented:
Hi all, I have submitted another commit.
I removed the `dask.config.set({'distributed.worker.daemon': False})` to be safe, as Emma mentioned that it might not be a good idea. I will follow up on this when the notebooks COP resumes, as James suggested.
I have also removed the commented lines about the default band names.
Effectively, the changes from the version in develop are the updated product and collection names in the function `feature_layers`. The warnings are still there, but at least the notebook runs with no errors.

robbibt reviewed (approved):
Thanks heaps @GL-S, I think this is a good approach for now. Any chance you could create quick Github issue in this repo describing the warnings issue, that we can refer to later to work out what's going wrong?

Other than that, I think this is ready to "Squash and merge" into the develop branch 🙂

GL-S commented:
Sure, I will create an issue now
